code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -280,14 +280,5 @@ Des: Removes all infected images from the alien.
 		if(A.update_remote_sight(src)) //returns 1 if we override all other sight updates.
 			return
 
-	for(var/obj/item/organ/internal/cyberimp/eyes/E in internal_organs)
```

Comment:
Yes, this means you can no longer augment the eyes of a Xeno---who ever did that anyway. 

These guys should probably have proper eyes, one day.